spec/lib/danger/helpers/comments_helper_spec.rb

```diff
@@ -275,29 +297,29 @@ class Dummy
   describe "#generate_comment" do
     it "no warnings, no errors, no messages" do
       result = dummy.generate_comment(warnings: [], errors: [], messages: [])
-      expect(result.gsub(/\s+/, "")).to eq(
+      expect(result.gsub(/\s+/, "")).to end_with(
```

Comment:
Not excited about this, but it was a pain to update those verifications after each change I made during implementing the summary. Most of those verifications should not really care about the summary? Matching on a regexp was also not something I wanted to do so I copped out with `end_with`.